YAML/syntax_test_sublime-syntax.sublime-syntax

```diff
@@ -0,0 +1,44 @@
+## SYNTAX TEST "Packages/User/scalaSublimeSyntax/sublime-syntax".sublime-syntax"
```

Comment:
This line should be `## SYNTAX TEST "YAML/sublime-syntax.sublime-syntax"`.

That is why your tests aren't being picked up.